<ui version="4.0" >
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Laue Preferences</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>390</y>
     <width>431</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>250</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Simulation</string>
   </property>
   <widget class="QCheckBox" name="LaueLorentzCheckBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Include Lorentz Pre-factor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LauePolnCheckBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Include Polarization Factor</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LaueSampleAbsCheckBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Include Sample Self Absorption</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title" >
    <string>Source</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text" >
     <string>Anode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LaueAnodeComboBox" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>30</y>
      <width>98</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LaueTubeVoltageSpinBox" >
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>70</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals" >
     <number>4</number>
    </property>
    <property name="maximum" >
     <double>100.000000000000000</double>
    </property>
    <property name="minimum" >
     <double>0.000000000000000</double>
    </property>
    <property name="value" >
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LaueLambdaMinSpinBox" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>140</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals" >
     <number>4</number>
    </property>
    <property name="maximum" >
     <double>100.000000000000000</double>
    </property>
    <property name="minimum" >
     <double>0.000000000000000</double>
    </property>
    <property name="value" >
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LaueLambdaMaxSpinBox" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>330</x>
      <y>110</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals" >
     <number>4</number>
    </property>
    <property name="maximum" >
     <double>100.000000000000000</double>
    </property>
    <property name="minimum" >
     <double>0.000000000000000</double>
    </property>
    <property name="value" >
     <double>40.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>Tube voltage (kV)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>Lambda Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>Lambda Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>180</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>Incident Polarization</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LaueOtherButton" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Other</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LaueXRayTubeButton" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>X-ray Tube</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
